target/linux/realtek/dts-5.10/rtl8382_iodata_bsh-g24mb.dts

```diff
@@ -0,0 +1,194 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "rtl838x.dtsi"
+
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+	compatible = "iodata,bsh-g24mb", "realtek,rtl838x-soc";
+	model = "I-O DATA BSH-G24MB";
+
+	aliases {
+		led-boot = &led_sys_loop;
+		led-failsafe = &led_sys_loop;
+		led-upgrade = &led_sys_loop;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,115200";
+	};
+
+	memory@0 {
+		device_type = "memory";
+		reg = <0x0 0x8000000>;
+	};
+
+	leds {
+		pinctrl-names = "default";
+		pinctrl-0 = <&pinmux_disable_sys_led>;
+		compatible = "gpio-leds";
+
+		led_sys_loop: led {
+			label = "red:sys_loop";
+			gpios = <&gpio0 0 GPIO_ACTIVE_HIGH>;
+			color = <LED_COLOR_ID_RED>;
+			function = LED_FUNCTION_STATUS;
+		};
+	};
+
+	keys {
+		compatible = "gpio-keys-polled";
+		poll-interval = <20>;
+
+		reset {
+			label = "reset";
+			gpios = <&gpio1 3 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+	};
+};
+
+&gpio1 {
```

Comment:
This does not compile for me, I am getting this error:
``
dir/target-mips_4kec_musl/linux-realtek_rtl838x/image-rtl8382_iodata_bsh-g24mb.dtb /home/hauke/openwrt/openwrt/build_dir/target-mips_4kec_musl/linux-realtek_rtl838x/image-rtl8382_iodata_bsh-g24mb.dtb.tmp
Error: ../dts-5.10/rtl8382_iodata_bsh-g24mb.dts:53.1-7 Label or path gpio1 not found
FATAL ERROR: Syntax error parsing input tree
``

Reply:
You have to do the same changes as done here: 4b31717fb031e6b99f476e79b5e8b3d6a8b0b9ff

Reply:
Oops, I forgot to update for it. I'll do...